tests/functional/duplicate_key_unlogged/test_duplicate_key_not_in_exceptions.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from dbt.tests.util import (
+    run_dbt,
+)
+
+_MODELS__view = """
+{{ config(
+    materialized='view',
+) }}
+
+with dupes as (
+    select 'foo' as key, 1 as value
+    union all
+    select 'foo' as key, 2 as value
+)
+
+select
+    object_agg(key, value)
+from dupes
+"""
+
+
+class TestRowValuesNotInExceptions:
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {"model.sql": _MODELS__view}
+
+    def test_row_values_were_scrubbed_from_duplicate_merge_exception(self, project):
+        result = run_dbt(["run", "-s", "model"])
```

Comment:
If you expect a dbt command to yield a non-zero exit status (i.e. fail / not pass), then you need to do something like [this](https://github.com/dbt-labs/dbt-snowflake/blob/903ccea98d354a7eaa7d3dfd13679dc29e4d4597/tests/functional/adapter/test_concurrency.py#L8) or else pytest will yield an error:

```suggestion
        result = run_dbt(["run", "-s", "model"], expect_pass=False)
```